### PR TITLE
Chg: update RTD to have only one Dockerfile

### PR DESCRIPTION
**Pull request by Omurshid**
DEVTOOLS-1338
refactor RTD to have only 1 Dockerfile.
Redis/ES will be configured to external services via Salt/Stacker

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,10 +1,20 @@
...
 
-ENV DEBIAN_FRONTEND=noninteractive
+ENV DEBIAN_FRONTEND=noninteractive 
+
+EXPOSE 9000
 RUN apt-get update && apt-get install -y tzdata \
     && ln -fs /usr/share/zoneinfo/America/Los_Angeles /etc/localtime \
     && dpkg-reconfigure --frontend noninteractive tzdata
 
+##Connect to redis cluster
```
Comment (goosemo) [inline Dockerfile]: remove, over comment. We have this history in `git`

File: Dockerfile [hunk 2]
```diff
@@ -54,6 +64,7 @@ RUN echo "0.0.0.0 docs.planfront.net" >> /etc/hosts
 
 ## This is a volume for our database
 VOLUME ["/persistent"]
-CMD ["./entrypoint.sh", "runserver", "0.0.0.0:80"]
+#this doesnt run fix it and ur good, i think lol
```
Comment (goosemo) [inline Dockerfile]: what does this mean? Does this need a comment to explain?

File: Jenkinsfile [hunk 3]
```diff
@@ -15,7 +15,18 @@ node {
         ])
     }
 
-    stage('build') {
-        sh "docker-compose build"
-    }
+    stage('Build Docker') {
```
Comment (goosemo) [inline Jenkinsfile]: You can call these like this now:
https://github.com/plangrid/plangrid-build-tools/blob/dev/vars/StandardBuild.groovy#L41

File: entrypoint.sh [hunk 4]
```diff
@@ -18,6 +18,8 @@ DB_LOCATION=${DB_LOCATION:-/persistent/dev.db}
 NEW_INSTALL=${NEW_INSTALL:-no}
 TEST_DATA=${TEST_DATA:-no}
 
+#connect to redis
```
Comment (goosemo) [inline entrypoint.sh]: remove comments, we'll have the history if we need to revert

File: Dockerfile [hunk 5]
```diff
@@ -1,10 +1,20 @@
 FROM ubuntu:18.04
```
Comment (goosemo) [inline Dockerfile]: Rename this to be `Dockerfile.j2`

File: local_settings.py.example [hunk 6]
```diff
@@ -1,69 +0,0 @@
-# Setting Overrides
```
Comment (goosemo) [inline local_settings.py.example:1]: Is this being removed because we're laying it down with Salt?

**Review by goosemo — CHANGES_REQUESTED** [review]

File: local_settings.py.example (hunk 6 above)
Comment (Omurshid) [reply local_settings.py.example:1]: yes

**Review by jpplangrid — CHANGES_REQUESTED** [review]
Use the GitClean and Checkout functions to get all the environmental variables you need to build (it's failing right now).

https://github.com/plangrid/plangrid-build-tools/blob/dev/vars/StandardBuild.groovy#L30-L32

Also, why not just use `StandardBuild()`?

**Comment by goosemo** [comment]
```
Step 8/20 : COPY readthedocs.org/requirements.txt ./
lstat readthedocs.org/requirements.txt: no such file or directory
```

**Pull request closed without merging**